src/ui/actions/timeline.ts

```diff
@@ -511,6 +511,19 @@ export function setFocusRegion(focusRegion: {
   return { type: "set_trim_region", focusRegion };
 }
 
+export function setFocusAroundTime(time: number, maxTime: number): UIThunkAction {
+  return (dispatch, getState) => {
+    const state = getState();
+    const zoomRegion = getZoomRegion(state);
+
+    const duration = zoomRegion.endTime;
```

Comment:
Technically, yes. `zoomRegion`'s an outdated concept that really should just be refactored out at some point. It used to be that scrolling vertically while on the timeline would zoom you in and out. That's since been taken out. So `zoomRegion.endTime` is always the end of the recording, and `zoomRegion.startTime` is always 0.

Reply:
My gut tells me that we should maybe still avoid hard-coding that dependency into places like this. So long as there's an "end time" attribute (and not a "duration" attribute) then it feels like it should be fair game to change the region's "start time" without expecting it to break other places like this.